src/app/main/component/events/components/events-list/events-list.component.ts

```diff
@@ -423,6 +423,19 @@ export class EventsListComponent implements OnInit, OnDestroy {
     return params;
   }
 
+  private getTypeFilter(list: string[]): string {
```

Comment:
should it be passed in the `this.selectedLocationFiltersList` parameters?
it has a global scope